cdc/processor.go

```diff
@@ -194,7 +194,7 @@ func newProcessor(
 		session:       session,
 		sink:          sink,
 		ddlPuller:     ddlPuller,
-		mounter:       entry.NewMounter(schemaStorage, changefeed.Config.Mounter.WorkerNum),
+		mounter:       entry.NewMounter(schemaStorage, changefeed.Config.Mounter.WorkerNum, changefeed.Config.Sink.OldValue),
```

Comment:
if the sink is mysql, `enableOldValue` should be always `false`

Reply:
MySQL sink can also benefit from `enableOldValue`, such as ignore the regions of index KV?